pkg/consts/consts.go

```diff
@@ -51,6 +55,7 @@ const (
 
 // Special label values of LabelRisingWaveGeneration.
 const (
+	// NoSync indicates that operator won't sync the resource after it's created.
```

Comment:
Which resources will it not sync?

Reply:
The resources labeled with `risingwave/generation=nosync`.